benchmarks/unstructured_parmetis.cpp

```diff
@@ -0,0 +1,793 @@
+﻿/* 
```

Comment:
This is another big file with some replication... Would it be better to split it into individual programs, or maybe reuse more code?

Reply:
I can do something here, sure. One thing could be to wrap the parmetis repartitioning section into something that might be useful for further tests / benchmarks. Also, I was thinking of generalizing e.g. the reindexing for the in-place receive in a future PR for the new Atlas bindings, since it could be also useful in general.